modules/nw-configuring-clb-timeouts.adoc

```diff
@@ -0,0 +1,42 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-clb-timeouts_{context}"]
+= Configuring Classic Load Balancer timeouts
+
+You can configure the default timeouts for a Classic Load Balancer (CLB) to extend idle connections.
```

Comment:
Is a CLB an API object or something else?

Reply:
Also, is it "AWS Classic Load Balancer" or just "Classic Load Balancer" when introduced in a module?

Reply:
it can be either, Classic Load Balancers are AWS-specific I was just trying to be consistent with the NLB content.